### PR TITLE
Documentation for mainnet

### PR DESCRIPTION
**Pull request by lezzokafka**
Check list: 
- [x] Getting `chain-maind` v1.1.0 binary;
- Other basic staking operation/ reward checking and collection ( could link to wallet "management" );
  - [x] Basic `bank send` tx
  - Staking operations --
    - [x] Delegation;
    - [x] Check reward
    - [x] Get reward;
    - [x] Unbond;
    - [x] Unjail.
 - [x] State-Sync
- [x] Check if its fully synced / node status ;
- [x] `create-validator`;
- ( when the nodes are ready) Tendermint/chain-maind config;
  - [x] Update seed nodes (subjected to change);
  - [x] Update genesis and its sha256sum checksum; 
-  ( when the nodes are ready) Final checking of links and nodes connection 
   - [x] Connecting public nodes for syncing ;
   - [x] Signature checking script 
   - [x] Public node with 26657 port

### CODE REVIEW

File: docs/getting-started/mainnet.md [hunk 1]
```diff
@@ -0,0 +1,469 @@
...
+  - 4-core, x86_64 architecture processor;
+  - 16 GB RAM;
+  - 1 TB of storage space.
+
+## Step 1. Get the Crypto.org Chain Mainnet binary
+
+
+
+::: tip Remarks:
+The following is the minimal setup to join Crypto.org Chain Mainnet. Furthermore, you may want to run full nodes as sentries (see [Tendermint](https://docs.tendermint.com/master/tendermint-core/running-in-production.html)), restrict your validator connections to only connect to your full nodes, test secure storage of validator keys etc.
```
Comment (calvinaco) [inline docs/getting-started/mainnet.md:26]:
```suggestion
The following is the minimal setup to join Crypto.org Chain Mainnet. Furthermore, you may want to run full nodes as sentries (see [Tendermint](https://docs.tendermint.com/master/tendermint-core/running-in-production.html)), restrict your validator connections to only connect to your full nodes, use secure storage and [key management](https://crypto.org/docs/getting-started/advanced-tmkms-integration.html) service for your validator keys etc.
```

File: docs/getting-started/mainnet.md [hunk 2]
```diff
@@ -0,0 +1,469 @@
...
+  ```
+##############################[TODO]- update seed,genesis - [TODO]##############################
+## Step 3. Run everything
+
+::: warning CAUTION
+This page only shows the minimal setup for validator node.
+
+In production, you may want to run full nodes
+as sentries (see [Tendermint](https://docs.tendermint.com/master/tendermint-core/running-in-production.html)), restrict your validator connections to only connect to your full nodes, test secure storage of validator keys with [tmkms](https://github.com/iqlusioninc/tmkms) etc.
+:::
```
Comment (calvinaco) [inline docs/getting-started/mainnet.md:156]: Duplicated as above? (below step 1)

File: docs/getting-started/mainnet.md [hunk 3]
```diff
@@ -0,0 +1,469 @@
...
+
+### Step 3-1. Create a new key and address
+
+Run the followings to create a new key. For example, you can create a key with the name `Default` by:
+
```
Comment (calvinaco) [inline docs/getting-started/mainnet.md:161]: I think here we should not suggest the use of chain-maind directly without any secure instructions. Maybe the better way is to redirect them to the mainnet address generation guide: https://crypto.org/docs/wallets/mainnet-address-generation.html

File: docs/getting-started/mainnet.md [hunk 4]
```diff
@@ -0,0 +1,469 @@
...
+--tendermint-url https://mainnet.crypto.org:26657 \
+--pubkey $(cat ~/.chain-maind/config/priv_validator_key.json | jq -r '.pub_key.value')
+
+The validator is in the active validator set under the address  <YOUR_VALIDATOR_ADDRESS>
+The validator is signing @ Block#<BLOCK_HEIGHT> 👍
+```
+
+
+
+Alternatively, you can run it on this [browser based IDE](https://repl.it/@allthatjazzleo/cryptocomcheckNodeJoinStatus#main.go), by specifying your validator public key in the `"YOUR_PUBKEY"` field, where this key can be obtained by running
```
Comment (calvinaco) [inline docs/getting-started/mainnet.md:323]: Is the script pointing to mainnet or testnet now?

**Review by calvinaco — COMMENTED** [review]
Overall looks good to me. Left some comments on the details.

File: docs/getting-started/mainnet.md (hunk 4 above)
Comment (lezzokafka) [reply docs/getting-started/mainnet.md:323]: I planed to remove this browser based IDE version of the signature checking script.

File: docs/getting-started/mainnet.md (hunk 3 above)
Comment (lezzokafka) [reply docs/getting-started/mainnet.md:161]: Nice suggestion :D

File: docs/getting-started/mainnet.md (hunk 2 above)
Comment (lezzokafka) [reply docs/getting-started/mainnet.md:156]: Nice catch 👍

File: docs/getting-started/mainnet.md (hunk 3 above)
Comment (lezzokafka) [reply docs/getting-started/mainnet.md:160]: Added, see this [link](https://github.com/crypto-org-chain/chain-docs/pull/139/commits/58b41340e73ddf740bc29edb2d74275013516641#diff-a556d8b640301c4c0d15e1f6cfb9b8b1d021a9d56d933eac621cab44a873802cR161)

File: docs/getting-started/mainnet.md (hunk 2 above)
Comment (lezzokafka) [reply docs/getting-started/mainnet.md]: [Removed](https://github.com/crypto-org-chain/chain-docs/pull/139/commits/58b41340e73ddf740bc29edb2d74275013516641#diff-a556d8b640301c4c0d15e1f6cfb9b8b1d021a9d56d933eac621cab44a873802cL151)

File: docs/getting-started/mainnet.md (hunk 1 above)
Comment (lezzokafka) [reply docs/getting-started/mainnet.md]: [Edited](https://github.com/crypto-org-chain/chain-docs/pull/139/commits/58b41340e73ddf740bc29edb2d74275013516641#diff-a556d8b640301c4c0d15e1f6cfb9b8b1d021a9d56d933eac621cab44a873802cL26)

**Pull request merged**